### PR TITLE
Andrew barker audio element edit 3338

### PR DESCRIPTION
**Pull request by AndrewJamesBarker**
### Description

Contributes to #<issue 3338>.  I added a descriptive picture of the audio player element. It is still possible to add more content here.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by AndrewJamesBarker** [comment]
I have the image pathway set to main to work when the merge is approved; hence, the image path is currently broken. You can review the image here: https://github.com/AndrewJamesBarker/docs/blob/andrew-barker-audio-element-edit-3338/media/html-audio-element-example.png
Thanks!

File: content/html/concepts/elements/terms/audio/audio.md [hunk 1]
```diff
@@ -46,5 +46,6 @@ The `<audio>` tag may include `<source>` tags and fallback text. It may also use
   <h3>Fallback text if audio will not play</h3>
 </audio>
 ```
-
 In the example above, the `<audio>` element contains the `controls` attribute to display user controls for the audio content such as a play and pause button. In addition, there are two `<source>` tags with different audio formats. If the browser doesn't support the `.ogg` file format, it will use the `.mp3` file. Lastly, the text that follows the `<source>` tags will display if the `<audio>` tag is not supported by the browser.
+
+![Audio Element](https://github.com/AndrewJamesBarker/docs/blob/main/media/html-audio-element-example.png)
```
Comment (yangc95) [inline content/html/concepts/elements/terms/audio/audio.md:51]:
```suggestion
![Audio Element](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-audio-element-example.png)
```

**Review by yangc95 — COMMENTED** [review]
🎯 @AndrewJamesBarker looks good to merge after the link is slightly adjusted!

**Comment by AndrewJamesBarker** [comment]
Thanks Christine, I updated to your suggestion :)

**Review by yangc95 — APPROVED** [review]

**Pull request merged**